templates/ORG/password-policy.yaml

```diff
@@ -1,10 +1,17 @@
 AWSTemplateFormatVersion: '2010-09-09'
 
+Parameters:
+  MinPasswordLength:
+    Default: 16
+    Description: The minimum length of a password
+    Type: Number
+    MinValue: 16
```

Comment:
Min is 16?  shouldn't this be a range that like min of 8 and max of 32 or something?

Reply:
Sure, I just put it to 16 to make sure we would not be able to change it to anything less than current min but then again we can change everything so...

Reply:
i see the problem @xschildw.  we can fix it in one of two ways, 
1) set MinPasswordLength to 8 for synapsedev account while keeping the others at 18 or 
2) change the default password in SSM in synapsedev to some value that’s 18 chars long.

i can do #2 for you if you like.

Reply:
I like option (2) best...